server/core/src/initServer.cpp

```diff
@@ -354,9 +355,10 @@ initRcatServerHostByFile() {
         for ( const auto& federation : irods::get_server_property<const nlohmann::json&> ( irods::CFG_FEDERATION_KW ) ) {
             try {
                 try {
-                    const auto fed_zone_key             = federation.at(irods::CFG_ZONE_KEY_KW).get<std::string>();
-                    const auto fed_zone_name            = federation.at(irods::CFG_ZONE_NAME_KW).get<std::string>();
-                    const auto fed_zone_negotiation_key = federation.at(irods::CFG_NEGOTIATION_KEY_KW).get<std::string>();
+                    const auto& fed_zone_key = boost::any_cast<const std::string&>(federation.at(irods::CFG_ZONE_KEY_KW));
+                    const auto& fed_zone_name = boost::any_cast<const std::string&>(federation.at(irods::CFG_ZONE_NAME_KW));
+                    const auto& fed_zone_negotiation_key = boost::any_cast<const std::string&>(federation.at(irods::CFG_NEGOTIATION_KEY_KW));
```

Comment:
This is a little questionable.

Reply:
because of the `boost::any`s?

Reply:
Mostly because we are iterating over an `nlohmann::json` object and using `boost::any_cast`. This is likely incorrect.

Reply:
done.